peps/pep-0728.rst

```diff
@@ -513,27 +598,31 @@ In this case, methods that are previously unavailable on a TypedDict are allowed
 
     reveal_type(not_required_num.popitem())  # OK. Revealed type is tuple[str, int]
 
-Open Issues
-===========
+How to Teach this
+=================
 
-Alternatives to the ``__extra__`` Reserved Key
-----------------------------------------------
+The choice of ``"__extra_items__"`` and the requirement of ``closed=True``
+whenever it is used as a special key intended to make it more understandable to
+new users even without former knowledge of this feature.
 
-As it was pointed out in the `PEP 705 review comment
-<https://discuss.python.org/t/pep-705-typeddict-read-only-and-other-keys/36457/6>`__,
-``__extra__`` as a reserved item has some disadvantages, including not allowing
-"__extra__" as a regular key, requiring special handling to disallow
-``Required`` and ``NotRequired``. There could be some better alternatives to
-this without the above-mentioned issues.
+Details of this should be documented in both the typing spec and the
+:mod:`typing` documentation.
 
 Backwards Compatibility
 =======================
 
-While dunder attributes like ``__extra__`` are reserved for stdlib, it is still
-a limitation that ``__extra__`` is no longer usable as a regular key. If the
-proposal is accepted, none of ``__required_keys__``, ``__optional_keys__``,
-``__readonly_keys__`` and ``__mutable_keys__`` should include ``__extra__`` in
-runtime.
+Because ``__extra_items__`` remains as a regular key if ``closed=True`` is not
+specified, no existing codebase will break due to this change.
+
+If the proposal is accepted, none of ``__required_keys__``,
+``__optional_keys__``, ``__readonly_keys__`` and ``__mutable_keys__`` should
+include ``"__extra_items__"`` defined on the same TypedDict type when
+``closed=True`` is specified.
+
+Note that ``closed`` as a keyword argument does not collide with the keyword
+arguments alternative to define keys with the functional syntax that allows
+things like ``TD = TypedDict("TD", foo=str, bar=int)``, because it is scheduled
+to be removed in Python 3.13 if support for ``closed`` is added.
```

Comment:
```suggestion
to be removed in Python 3.13.
```
It's scheduled to be removed regardless of this PEP.

However, it's true that this *does* cause a minor issue for typing-extensions. Probably fine to omit discussing that in this PEP though.